clang/test/Driver/clang_f_opts.c

```diff
@@ -605,7 +605,7 @@
 // RUN: %clang -### -S -fjmc -g -fno-jmc -target x86_64-pc-windows-msvc %s 2>&1 | FileCheck -check-prefix=CHECK_NOJMC %s
 // RUN: %clang -### -S -fjmc -g -target x86_64-unknown-linux %s 2>&1 | FileCheck -check-prefix=CHECK_JMC %s
 // RUN: %clang -### -S -fjmc -g -fno-jmc -target x86_64-unknown-linux %s 2>&1 | FileCheck -check-prefix=CHECK_NOJMC %s
-// RUN: %clang -### -fjmc -g -flto -target x86_64-pc-windows-msvc %s 2>&1 | FileCheck -check-prefix=CHECK_NOJMC_LTO %s
+// RUN: %clang -### -fjmc -g -flto -fuse-ld=lld -target x86_64-pc-windows-msvc %s 2>&1 | FileCheck -check-prefix=CHECK_NOJMC_LTO %s
```

Comment:
`-fuse-ld=lld` would lead to a failure if lld is not found in program paths. This failure happens on Linux, not sure about Windows.

Reply:
Ahh the Windows MSVC linker code uses `GetProgramPath` and not `GetLinkerPath` so this error won't happen on the windows-msvc target. I'll see if there is a better way to do this anyhow.

Reply:
I did some more digging here. Unit tests testing `-fuse-ld=lld` for `*-windows-msvc` already assumes that we won't error if `lld-link.exe` cannot be found. We would have to fix quite a few unit tests if the `*-windows-msvc` target ever starts erroring if `lld-link.exe` cannot be found. Example, https://github.com/llvm/llvm-project/blob/main/clang/test/Driver/cl-link.c#L64.